phpBB/phpbb/install/module_base.php

```diff
@@ -110,10 +110,38 @@ public function run()
 
 		if ($task_index < $iterator->count())
 		{
-			$iterator->seek($task_index);
+		    // Receiving task index
```

Comment:
Would no the increase of the execution time be insignificant since ``` current() ``` is used already many times in similar situations? Moreover, is not it necessary to come to the previous step for this solution? It seems to me that this approach is the only way to return to the previous step.

So you mean that when the user returns to a previous step, the previously inputted information by the user should be auto included back? If so, I agree with the approach and would like to try to implement it. Actually, I already tried to do it, but it turned out to be more difficult and I have not done it yet.